docs/client-library.md

```diff
@@ -0,0 +1,28 @@
+# HIL Client Library
+
+##Description
+The HIL API Client Library for Python is designed for Python client-application developers. It offers simple access to HIL APIs.
+
+## How to Install HIL Modules?
+```
+pip install git+https://github.com/cci-moc/hil
+```
+
```

Comment:
We might want to document which client library commands are available if we don't want users to have to look through the source.